packages/flutter/lib/src/material/text_field.dart

```diff
@@ -635,10 +636,21 @@ class TextField extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.cursorRadius}
   final Radius cursorRadius;
 
-  /// The color to use when painting the cursor.
+  /// The color used to paint the cursor.
```

Comment:
We seem to be on a nice trajectory here :-). How about : The color of the cursor. Here and elsewhere and in general.  For example: The color of the TextField's selection handles, instead of "The color used to paint the selection handles on the text field".